addons/product/models/uom_uom.py

```diff
@@ -8,14 +8,6 @@ class UomUom(models.Model):
     _inherit = 'uom.uom'
 
     product_uom_ids = fields.One2many('product.uom', 'uom_id', string='Barcodes', domain=lambda self: ['|', ('product_id', '=', self.env.context.get('product_id')), ('product_id', 'in', self.env.context.get('product_ids', []))])
-    packaging_barcodes_count = fields.Integer('Packaging Barcodes', compute='_compute_packaging_barcodes_count')
```

Comment:
forbiden. Just return directly in the compute and remove it in master

Reply:
Even if it's a non-stored field?